azure-pipelines.yml

```diff
@@ -102,6 +102,40 @@ jobs:
         ./scripts/ci/verify_modified_index.sh
       displayName: "CLI Linter on Modified Extension"
 
+- job: LintModifiedExtensionsNew
+  displayName: "CLI Linter on Modified Extensions (New)"
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+    - task: UsePythonVersion@0
+      displayName: 'Use Python 3.6'
+      inputs:
+        versionSpec: 3.6
+    - bash: |
+        set -ev
+
+        # prepare and activate virtualenv
+        pip install virtualenv
+        python -m virtualenv venv/
+        source ./venv/bin/activate
+
+        # clone azure-cli
+        git clone --single-branch -b dev https://github.com/Azure/azure-cli.git ../azure-cli
```

Comment:
Is it more stable to use release instead of dev?

Reply:
discussed before with @Juliehzl, align with the most advance feature on dev give user the ability to test changes on both azure-cli and azure-cli-extensions, and discover potential problems ASAP.

Reply:
Well. Indeed. But any issue in auzre-cli will block the CI in azure extension.

Reply:
If any issue can block CI of azure-cli-extensions, then let it break. 
It's a trade-off. Do you think we should change it?